lib/client/src/client.rs

```diff
@@ -1963,7 +2209,7 @@ impl MangoClient {
 #[derive(Debug, thiserror::Error)]
 pub enum MangoClientError {
     #[error("Transaction simulation error. Error: {err:?}, Logs: {}",
-        .logs.iter().join("; ")
+    .logs.iter().join("; ")
```

Comment:
this one doesn't look nice - did cargo fmt do it?

Reply:
yes it's cargo fmt